Header                       header                 # The message of header

DetectionObjectInfo[128]        detection_object_info  # The information of detection object

uint16                       detection_object_num   # The number of detection object

TrackingObjectInfo[128]         tracking_object_info   # The information of tracking object

uint16                       tracking_object_num    # The number of tracking object